### PR TITLE
feat(StatusBar): Add a status bar to nteract

### PR DESCRIPTION
**Pull request by captainsafia**
I was thinking of having an isModified indicator in there. Any other things people would like to see in the status bar? I think we were also talking about having kernel name/status on there too.

screen shot 2016-09-20 at 8 40 57 pm
screen shot 2016-09-20 at 8 40 52 pm

### CODE REVIEW

**Comment by captainsafia** [comment]
So I'm not the greatest with CSS and there is a problem with the float on the saved value in the status bar. Does anyone know why it might be happening? I've already tried tinkering with clearfixes.

screen shot 2016-09-21 at 12 17 50 pm

**Comment by lgeiger** [comment]
That's likely a issue with the padding. If you like I can push a fix to this branch.

**Comment by captainsafia** [comment]
> That's likely a issue with the padding. If you like I can push a fix to this branch.

Yes, please! Thanks, Lukas!

**Comment by lgeiger** [comment]
bildschirmfoto 2016-09-22 um 15 55 25

**Comment by captainsafia** [comment]
Yay! This can be merged now.

Also, I'm not entirely sure what is up with CodeCov but I've got an updated master and running `npm run coverage` on my local returns a coverage of 81.75% not 81.55%. ¯_(ツ)_/¯

**Comment by rgbkrk** [comment]
That's looking goooood.

File: src/notebook/components/status-bar.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+export default class StatusBar extends React.Component {
+  static propTypes = {
+    notebook: React.PropTypes.any,
+    lastSaved: React.PropTypes.instanceOf(Date),
+    kernelSpecName: React.PropTypes.string,
+  };
+
+  shouldComponentUpdate(nextProps) {
+    if (this.props.notebook !== nextProps.notebook ||
+        this.props.lastSaved !== nextProps.lastSaved) {
```
Comment (rgbkrk) [inline src/notebook/components/status-bar.js:13]: :+1:

File: src/notebook/components/status-bar.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      <div className="status-bar">
+        <span className="pull-right">
+        { this.props.lastSaved ?
+          <p> Last saved {moment(this.props.lastSaved).fromNow()} </p> :
+          <p> Not saved yet </p>
+        }
+        </span>
+        <span className="pull-left">
+          <p>nteract | {this.props.kernelSpecName}</p>
+        </span>
```
Comment (rgbkrk) [inline src/notebook/components/status-bar.js:30]: :)

File: src/notebook/middlewares.js [hunk 3]
```diff
@@ -4,49 +4,11 @@ import { createEpicMiddleware, combineEpics } from 'redux-observable';
...
-
-  return next(action);
-};
 
 const rootEpic = combineEpics(...epics);
 
 const middlewares = [
   createEpicMiddleware(rootEpic),
-  triggerModified,
 ];
```
Comment (rgbkrk) [inline src/notebook/middlewares.js:13]: Woo!

**Review by rgbkrk — APPROVED** [review]

**Comment by rgbkrk** [comment]
I'll check out what's up with coverage before merging. Excellent stuff here. This means we can go back to the native document title, which comes complete with a few other niceties.

**Comment by rgbkrk** [comment]
Absolute coverage decreased but relative coverage increased. This is due to how much code was here before that was tested (yet way verbose).

**Pull request merged**